_config.yml

```diff
@@ -19,14 +19,14 @@
 # in the templates via {{ site.myvariable }}.
 
 title: " <b>Engineering</b> Blog"
-logo: "/assets/images/devo-theme/devo-logo-dark.svg"
+logo: "assets/images/devo-theme/devo-logo-dark.svg"
 locale: "en"
 #subtitle: "whatever"
 name: Devo Engineering Blog
 #email: your-email@example.com
 description: >- # this means to ignore newlines until "baseurl:"
-baseurl: "" # the subpath of your site, e.g. /blog
-url: "https://devoinc.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+baseurl: "/devoinc.github.io" # the subpath of your site, e.g. /blog
+url: "https://tripu.github.io" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
I guess we need to change that before merging this ;)

Reply:
You just said that on the PR :facepalm:

Reply:
:blush: